src/KMonad/App/Parser/TokenJoiner.hs

```diff
@@ -352,6 +352,8 @@ joinButton ns als =
     -- Various compound buttons
     KComposeSeq bs     -> view cmpKey >>= \c -> jst $ tapMacro . (c:) <$> mapM go bs
     KTapMacro bs       -> jst $ tapMacro           <$> mapM go bs
+    KTapMacroRelease bs
+      -> jst $ tapMacroRelease            <$> mapM go bs
```

Comment:
In order to be consistent with the rest of the code base, format this as either

``` haskell
    KTapMacroRelease bs-> jst $ tapMacroRelease            <$> mapM go bs
```
or
``` haskell
    KTapMacroRelease bs ->
      jst $ tapMacroRelease <$> mapM go bs
```